metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -220,9 +221,22 @@ object UserConfiguration {
           |See https://scalameta.org/metals/docs/contributors/remote-language-server.html for
           |documentation on remote language servers.
           |""".stripMargin
+      ),
+      UserConfigurationOption(
```

Comment:
I would not want to include it as a user config options, as most people will have no idea what this is about and will not need to change it. Could you add it as another option in `MetalsServerConfig` ?

Reply:
If this config option is in UserConfiguration user could change property in ~/.config/Code/User/settings.json (for example) oneself (or from IDE settings page - i think it's target solution), restart build server and continue work
If this config option is in MetalsServerConfig which setting file should be changed and who could change it?

Reply:
You can add `-Dmetal-bloop-port=123` in `metals.serverProperties`, which will add the java property to the running server process.

Reply:
Done